Dart/src/com/jetbrains/lang/dart/ide/runner/server/vmService/DartVmServiceBreakpointHandler.java

```diff
@@ -116,3 +104,70 @@ public void breakpointFailed(@NotNull final XLineBreakpoint<XBreakpointPropertie
     return myVmBreakpointIdToXBreakpointMap.get(vmBreakpoint.getId());
   }
 }
+
+class IsolateBreakpointInfo {
+  private final String myIsolateId;
+  private final DartVmServiceDebugProcess myDebugProcess;
+  List<String> myTemporaryBreakpoints = new ArrayList<>();
```

Comment:
Since we have a lot of different entities that have 'breakpoint' word, we can clarify field to be `myTemporaryVmBreakpointIds`.